COPY/usr/bin/manageiq-messaging-ready

```diff
@@ -28,17 +28,18 @@ def manageiq_msging_ready?(msg_yaml, env = "production")
   options[:password] = ManageIQ::Password.try_decrypt(options[:password]) if options[:password]
   options[:client_ref] = "manageiq-messaging-ready"
 
-  # Test broker connection by publishing message to queue and immediately consuming message
+  # Test broker connection by listing topics
   broker = ManageIQ::Messaging::Client.open(options)
-  broker.publish_message(:service => "manageiq.liveness-check", :message => "test message", :payload => {})
-  broker.subscribe_messages(:service => "manageiq.liveness-check") { break }
+  broker.topics
 rescue => err
   puts err
   puts "Kafka is not ready yet"
   false
 else
   puts "Kafka is up and running"
   true
+ensure
+  broker.close
```

Comment:
Just in case something before `broker = ManageIQ::Messaging::Client.open(options)` raises an exception like `ManageIQ::Password`, best to handle `nil` here
```suggestion
  broker&.close
```